Vagrantfile.example

```diff
@@ -9,6 +9,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # explicitly mount the code using NFSv4.
  config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_version: 4, nfs_udp: false
 
+ # Uncomment this line if you would like the guest VM to be automatically updated
```

Comment:
A blank line would be good here to help highlight the code line that is meant to be uncommented in the text